test/scss/value/test.coffee

```diff
@@ -1,7 +1,15 @@
 describe 'scss/value >>', ->
 
-  # it '0', -> this.shouldBeOk()
-  # it '1', -> this.shouldBeOk()
-  # it '2', -> this.shouldBeOk()
-  # it '3', -> this.shouldBeOk()
```

Comment:
Can you please use `it.skip` instead of commenting in future? I usually grep for `skip` to see if there are any tests to fix.